heron/executor/src/python/BUILD

```diff
@@ -2,8 +2,15 @@ package(default_visibility = ["//visibility:public"])
 
 load("/tools/rules/pex_rules", "pex_library", "pex_binary")
 
+pex_library(
+    name = "heron-executor-lib",
+    srcs = ["heron_executor.py"],
+    deps = ["//heron/statemgrs/src/python:zk-statemgr-py"],
+    reqs = ["pyyaml==3.10"],
+)
+
 pex_binary(
     name = "heron-executor",
-    srcs = ["heron-executor.py"],
-    reqs = ["pyyaml==3.10"],
+    srcs = ["heron_executor.py"],
+    deps = [":heron-executor-lib"],
```

Comment:
yeah, this is a little awkward but we need `pex_library` since `pex_test` takes that as a dep, but can't test against a `pex_binary`. And both `pex_library` and `pex_binary` require `srcs` so we have to declare `heron_executor.py` in both.